### PR TITLE
Add "set contract version" to migrate methods 

### PR DESCRIPTION
**Pull request by blue-note**
I know how we do the versioning will change with [this PR](https://github.com/DA0-DA0/dao-contracts/pull/461), but let's also make sure we update the contract version in the migrate methods. This will enable us to start testing v2 contracts from frontend as well (checks the contract version)

### CODE REVIEW

File: contracts/cw721-stake/src/contract.rs
Comment (ezekiiel) [inline contracts/cw721-stake/src/contract.rs:535]: How would you feel about adding some really simple tests for these? Could probably just write one like (pseudocode):

```
let mut deps = mock_dependencies();
cw2::set_contract_version(deps.storage, "mock", "mock").unwrap();
migrate(deps.as_mut(), mock_env(), MigrateMsg {}).unwrap();
let version = cw2::get_contract_version(deps.storage).unwrap();
assert_eq!(version.version, env!("CARGO_PKG_VERSION");
```

and copy it into all of the contracts. would catch if we forget about this again.

**Review by ezekiiel — COMMENTED** [review]
Great catch!!

**Review by ezekiiel — APPROVED** [review]
Approving so you're not blocked on another review from me. :)

File: contracts/cw721-stake/src/contract.rs
Comment (blue-note) [reply contracts/cw721-stake/src/contract.rs:535]: yess good call

File: types/yarn.lock
Comment (de-husk) [inline types/yarn.lock:10]: Looks like these `types/yarn.lock` changes were accidentally added through find and replace. We should remove these.

**Review by de-husk — CHANGES_REQUESTED** [review]

File: types/yarn.lock
Comment (blue-note) [reply types/yarn.lock:10]: whoops!

File: ci/integration_tests/Cargo.toml
Comment (de-husk) [inline ci/integration_tests/Cargo.toml:22]: Thanks for fixing this! Now that we are actually versioning the smart contracts, the versions here should all be the latest.

**Review by de-husk — APPROVED** [review]

**Pull request merged**
